engine/src/main/java/org/operaton/bpm/engine/history/HistoricProcessInstanceQuery.java

```diff
@@ -162,6 +162,13 @@ public interface HistoricProcessInstanceQuery extends Query<HistoricProcessInsta
    */
   HistoricProcessInstanceQuery incidentMessageLike(String incidentMessageLike);
 
+  /**
+   * Only select historic process instances which are associated with jobs that have exceptions and retries left.
+   *
+   * @return HistoricProcessInstanceQuery
+   */
+  HistoricProcessInstanceQuery withJobsRetrying();
```

Comment:
for backward compatibility adding methods to a public API interface should be done with a _default method_.